### ISSUE TITLE
Githhub-pages site should include both Python and R package docs

### ISSUE THREAD

**Issue by jameslamb** [UptakeOpenSource/uptasticsearch#131]
Right now, we are building our Github Pages site off the `docs/` folder on `master`. That only includes the `pkgdown` site for the R package.

Thanks to @csyhuang on #130 , the Python package will soon have Sphinx docs.

We should have a strategy for hosting Python and R docs together on https://uptakeopensource.github.io/uptasticsearch/

One strategy could be to write a custom `index.html` with links down to python-specific and R-specific folders and then to add something like this to `Makefile`:

```
site: docs_r docs_python
    mv r-pkg/docs docs/r/
    mv py-pkg/docs/built docs/py
```

But totally open to other suggestions! Also note that our current strategy for `pkgdown` has been to host all the files checked into the repo on master, but surely there must be a better way.

**Comment by jayqi** [comment]
Yes please. Also the URL to the docs site is not in the repo description as is convention.

**Comment by jameslamb** [comment]
The link in the original write-up for this issue is broken... the docs are now hosted at https://uptake.github.io/uptasticsearch/

As of #224, this project no longer contains a Python package. So this issue can be closed.
